data/scripts/actions/christmas_bundle.lua

```diff
@@ -0,0 +1,43 @@
+local christmasBundle = Action()
+
+function christmasBundle.onUse(player, item, fromPosition, target, toPosition, isHotkey)
+	local presents = {
+		[6507] = { -- red christmas bundle
+			{6569, 15}, {2687, 20}, {2688, 10}, {2675, 10}, {2674, 5}, 6501, 6502, 6490, 6504, 6388
+		},
+		[6508] = { -- blue christmas bundle
+			{6569, 15}, {2687, 20}, {2688, 10}, {2675, 10}, {2674, 5}, 6501, 6502, 6490, 6505, 6388
+		},
+		[6509] = { -- green christmas bundle
+			{6569, 15}, {2687, 20}, {2688, 10}, {2675, 10}, {2674, 5}, 6501, 6502, 6490, 6503, 6388
+		}
+	}
```

Comment:
this table looks repeatable. I think this should be flat cuz you don't need key once you have ids registration on end of the script and the values are the same for all keys.
what are you think about that?

Reply:
yes but the code will look even messier because of this I guess, [look](https://otland.net/threads/tfs-1-3-how-to-make-script-select-more-than-one-reward.268511/#post-2593711)

Reply:
```lua
local rewards = {
  {6569, 15}, {2687, 20}, {2688, 10}, {2675, 10}, {2674, 5}, 6501, 6502, 6490, 6504, 6388
}
```
messier? lol... well, i just saying to make the table simple to this cuz all rewards is the same

Reply:
The rewards are not all the same. Mind 6503 through 6505.

Reply:
that's right, sorry :P